AirCasting/Utils/Strings.swift

```diff
@@ -15,23 +15,52 @@ struct Strings {
         static let about: String = "About AirCasting"
     }
     
-    struct BackendSettings {
+    enum BackendSettings {
         static let backendSettings: String = "Backend settings"
         static let Ok: String = "Ok"
         static let Cancel: String = "Cancel"
     }
     
-    struct MyAccountSettings {
+    enum MyAccountSettings {
         static let title: String = "My account"
         static let logStatus: String = "You aren’t currently logged in"
         static let notLogged: String = "You aren’t currently logged in"
         static let createAccount: String = "Create an account"
         static let logIn: String = "Log In"
     }
     
-    struct SignOutSettings {
+    enum SignOutSettings {
         static let title: String = "My account"
         static let Logged: String = "You are currently logged in as admin@admin.com"
         static let signOut: String = "Sign Out"
     }
+    
+    enum ForgotPassword {
+        static let title = "Forgot Password"
+        static let description = "You will get en email with details after 'send new' button pressed"
```

Comment:
I think this is unnecessary, Android doesn't have that. And they have two buttons "Ok" and "Cancel". Why did you decide to do it differently? ;)